www/common/outer/async-store.js

```diff
@@ -1220,7 +1221,20 @@ define([
                 if (!s.id) { inMyDrive = res.length; }
 
                 Array.prototype.push.apply(allData, res);
+
+                if (res && res[0]) {
+                    var elementPath = store.manager.user.userObject.findFile(res[0].id)[0];
+                    if (elementPath) {
+                        path.push(elementPath);
+                    }
+                }    
+
             });
+
+            if (path[0]) {
```

Comment:
I don't really understand why `path` should be an array in you only use the first entry